packages/dds/tree/src/forest/cursorLegacy.ts

```diff
@@ -73,7 +73,7 @@ export interface ITreeCursor<TResult = TreeNavigationResult> {
     /**
      * @returns the number of immediate children for the given key of the currently selected node.
      */
-    length(key: FieldKey): number;
+    length(key?: FieldKey): number;
```

Comment:
If you are going to change the API here, you need to update the documentation and tests accordingly.

If no key is provided, it looks like you are returning the number of siblings of the current node?
This seems like a reasonable change to make. You might also an extra feature in the new cursor API to handle whatever needed this change as well.